src/main/java/io/github/yamlpath/processor/expressions/IsEqualExpressionProcessor.java

```diff
@@ -21,6 +21,8 @@ public boolean evaluate(String left, String right, Map<Object, Object> resource)
         Object value = parser.readSingle(left);
         if (value instanceof Integer) {
```

Comment:
Shouldn't that be `Number` instead of `Integer` to also support non-integers?

Reply:
Fair enough. This was not as simple of using Number, but it's a fair request. I've just updated with supporting any kind of numbers, plus adding other operators.